sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -306,13 +457,63 @@ export interface IssuerProperties {
   /**
    * When the issuer was created.
    */
-  created?: Date;
+  createdOn?: Date;
   /**
    * When the issuer was updated.
    */
-  updated?: Date;
+  updatedOn?: Date;
   /**
    * Name of the issuer
    */
   name?: string;
 }
+
+/**
+ * @interface
+ * An interface representing optional parameters for CertificateClient paged operations passed to {@link listCertificates}.
+ */
+export interface ListCertificatesOptions
+  extends KeyVaultClientGetCertificatesOptionalParams,
+    coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing optional parameters for CertificateClient paged operations passed to {@link listCertificateVersions}.
+ */
+export interface ListCertificateVersionsOptions
+  extends KeyVaultClientGetCertificatesOptionalParams,
+    coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing optional parameters for CertificateClient paged operations passed to {@link listIssuers}.
+ */
+export interface ListIssuersOptions
+  extends KeyVaultClientGetCertificateIssuersOptionalParams,
+    coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing optional parameters for CertificateClient paged operations passed to {@link listDeletedCertificates}.
+ */
+export interface ListDeletedCertificatesOptions
+  extends KeyVaultClientGetDeletedCertificatesOptionalParams,
+    coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing optional parameters for CertificateClient paged operations passed to {@link mergeCertificate}.
+ */
+export interface MergeCertificateOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing optional parameters for CertificateClient paged operations passed to {@link recoverDeletedCertificate}.
+ */
+export interface RecoverDeletedCertificateOptions extends coreHttp.OperationOptions {}
```

Comment:
Recovering deleted certificate is not a paged operation right? Same for `mergeCertificate` and `restoreCertificateBackup`
If so, then the description for all 3 needs update